example/pom.xml

```diff
@@ -47,6 +47,7 @@
         <module>rest-java-example</module>
         <module>flink-sql</module>
         <module>schema</module>
+        <module>count-point-processor</module>
```

Comment:
```suggestion
        <module>pipe-count-point-processor</module>
```

Reply:
also change name from opc-ua-sink to pipe-opc-ua-sink

Reply:
done.